store/store.go

```diff
@@ -371,6 +371,8 @@ type PostStore interface {
 	GetOldestEntityCreationTime() (int64, error)
 	HasAutoResponsePostByUserSince(options model.GetPostsSinceOptions, userId string) (bool, error)
 	GetPostsSinceForSync(options model.GetPostsSinceForSyncOptions, cursor model.GetPostsSinceForSyncCursor, limit int) ([]*model.Post, model.GetPostsSinceForSyncCursor, error)
+	// GetRecentNthPostTime returns CreateAt time of most recent nth user-post
```

Comment:
`// GetNthRecentPostTime returns the CreateAt time of the nth most recent post.` and rename the method name as well.

Reply:
You just changed the comment, but the method needs to be renamed as well :)

Reply:
Oops😅